piano/app/src/main/res/layout/fragment_white_piano_key.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="wrap_content"
+    android:layout_height="match_parent"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    tools:context=".WhitePianoKeyFragment">
+
+    <Button
+        android:id="@+id/whitePianoKeyButton"
```

Comment:
Godt valg av id-navn for knappen. Synes det er fint at du har inkludert button i navnet så er mer informerende hva elementet egentlig er.